packages/ckeditor5-build-multi-root/src/ckeditor.ts

```diff
@@ -57,11 +61,12 @@ import {
 import { TextTransformation } from '@ckeditor/ckeditor5-typing';
 import { TrackChanges, TrackChangesData } from '@ckeditor/ckeditor5-track-changes';
 import { Undo } from '@ckeditor/ckeditor5-undo';
-import WProofreader from '@webspellchecker/wproofreader-ckeditor5/src/wproofreader';
+import { AIAssistant, OpenAITextAdapter } from '@ckeditor/ckeditor5-ai';
```

Comment:
@lucasfontes2604 wondering if there's a mechanism to move some of this code into the authorium repo so that we can implement `class CustomOpenAITextAdapter extends OpenAITextAdapter` on that side of the wire to customize the `prepareMessages` and `sendRequest` methods to have better control over the payloads we are sending / receiving in the editor ui (i.e. prompt customization etc.)